examples/cpp/grpc/pingpong/pingpong.cpp

```diff
@@ -149,8 +143,15 @@ void assertResponseContents(const bond::ext::gRPC::unary_call_result<PingReply>&
     }
 }
 
-void assertResponseContents(const bond::ext::gRPC::unary_call_result<bond::Void>&, size_t)
-{}
+void assertResponseContents(const bond::ext::gRPC::unary_call_result<void>& result, size_t)
+{
+    (void)result.context(), result.status();
```

Comment:
what is this syntax doing?

Reply:
Ignore result and make sure it compiles.